source/guides/get-help-from-other-roles.html.md

```diff
@@ -23,10 +24,10 @@ You can use a few pieces of information to help other people understand, assess
 
 ## Product and delivery managers
 
-A product manager is responsible for the quality of their products. They use their knowledge of user needs and business goals to frame problems and set priorities for delivery teams. A delivery manager is accountable for the delivery of products and services. You can read more about [product managers' skills](https://ddat-capability-framework.service.gov.uk/product-manager.html) and [deliver managers' skills](https://ddat-capability-framework.service.gov.uk/delivery-manager.html) on the Digital, Data and Technology Profession Capability Framework.
+A product manager is responsible for the quality of their products. They use their knowledge of user needs and business goals to frame problems and set priorities for delivery teams. A delivery manager is accountable for the delivery of products and services. You can read more about [product managers' skills](https://ddat-capability-framework.service.gov.uk/product-manager.html) and [deliver managers' skills](https://ddat-capability-framework.service.gov.uk/delivery-manager.html) on the Government Digital and Data Profession Capability Framework.
```

Comment:
```suggestion
A product manager is responsible for the quality of their products. They use their knowledge of user needs and business goals to frame problems and set priorities for delivery teams. A delivery manager is accountable for the delivery of products and services. You can read more about [product managers' skills](https://ddat-capability-framework.service.gov.uk/product-manager.html) and [deliver managers' skills](https://ddat-capability-framework.service.gov.uk/delivery-manager.html) on the Digital, Data and Technology Profession Capability Framework.
```

I know the profession has changed its name but the framework hasn't.

Reply:
It seems like it has? The name given on the webpage is "Government Digital and Data Profession Capability Framework"

Reply:
Oh wow, so it has! Sorry beeps, I was acting off of old information.